config/default.yml

```diff
@@ -28,6 +28,10 @@ linters:
     enabled: true
     max: 80
 
+  FileLength:
+    enabled: true
+    max: 300
```

Comment:
This default settings is difficult to decide.

Reply:
Given the difficulty of deciding this, and also due to the fact that this a highly personal/contentious decision for a codebase, we should have the default configuration be `enabled: false`.

Reply:
OK. I see.